hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCreateTransitionLogic.java

```diff
@@ -161,6 +161,9 @@ public void doStateTransitionOperation(
 
 		// --- Do the business logic ---
 		ContractCustomizer hapiSenderCustomizer = fromHapiCreation(key, consensusTime, op);
+		if (!properties.areContractAutoAssociationsEnabled()) {
```

Comment:
Don't this need to be done in `UpdateCustomizerFactory.customizerFor` as well ?

Reply:
Resolved in other PR. This PR will be closed.

Reply:
@Neeharika-Sompalli That was against `release/0.26` which won't be otherwise merged to `master`; fixed here also.

Reply:
Got it !Thanks Michael !